test/services/jobs/notifications/update-event-error-count.service.test.js

```diff
@@ -0,0 +1,129 @@
+'use strict'
+
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+
+const { describe, it, beforeEach } = (exports.lab = Lab.script())
+const { expect } = Code
+
+// Test helpers
+const EventHelper = require('../../../support/helpers/event.helper.js')
+const EventModel = require('../../../../app/models/event.model.js')
+const ScheduledNotificationHelper = require('../../../support/helpers/scheduled-notification.helper.js')
+
+// Thing under test
+const UpdateEventErrorCountService = require('../../../../app/services/jobs/notifications/update-event-error-count.service.js')
+
+describe('Job - Notifications - Update event service', () => {
+  let event
+
+  beforeEach(async () => {
+    event = await EventHelper.add({
+      type: 'notification',
+      status: 'completed',
+      metadata: {}
+    })
+
+    await ScheduledNotificationHelper.add({
+      eventId: event.id,
+      status: 'sending'
+    })
+  })
+
+  describe('when there are errors', () => {
+    beforeEach(async () => {
+      await ScheduledNotificationHelper.add({
+        eventId: event.id,
+        status: 'error'
+      })
+    })
+
+    describe('and the "scheduledNotifications" have a single error', () => {
+      it('updates the error count"', async () => {
+        await UpdateEventErrorCountService.go([event.id])
+
+        const result = await EventModel.query().findById(event.id)
```

Comment:
Will this not work?

I appreciate it's the same result, but IMHO it makes it explicit we testing that the event we created got updated by the service.

```suggestion
        const refreshEvent = await event.$query()
```

(Same suggestion applies to all applicable tests).

Reply:
I keep forgetting this.